modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/EnrollmentService.java

```diff
@@ -97,9 +75,12 @@ public Message isEnrolled(@PathParam("id") String id) throws AndroidAgentExcepti
 			result = AndroidAPIUtils.getDeviceManagementService().isEnrolled(deviceIdentifier);
 			if (result) {
 				responseMsg.setResponseMessage("Device has already enrolled");
+				responseMsg.setResponseCode(Response.Status.ACCEPTED.toString());
```

Comment:
Is this the HTTP status code? Can you please explain why we use toString() method?
Thanks